game-app/game-core/src/main/java/org/triplea/util/PointFileReaderWriter.java

```diff
@@ -198,18 +186,18 @@ public static void writeOneToManyPlacements(
                         + " | overflowToLeft="
                         + entry.getValue().getSecond())
             .collect(Collectors.joining("\r\n")),
-        sink);
+        output);
   }
 
   /** Returns a map of the form String -> (points, overflowToLeft). */
-  public static Map<String, Tuple<List<Point>, Boolean>> readOneToManyPlacements(
-      final InputStream stream) throws IOException {
-    checkNotNull(stream);
+  public static Map<String, Tuple<List<Point>, Boolean>> readOneToManyPlacements(final Path input)
+      throws IOException {
+    checkNotNull(input);
 
     final Map<String, List<Point>> mapping = new HashMap<>();
     final Map<String, Tuple<List<Point>, Boolean>> result = new HashMap<>();
```

Comment:
Would the util method: UrlStreams#openStream help us at all in any of this?

Reply:
I don't think urlstreams has any functionality regarding paths, but i didn't check

Reply:
It would be reasonable to add some overloads to it if it's needed, overall though we may be able to avoid the need for paths and files altogeher. UrlStreams#openStreams opens and reads a URL stream and takes a function that process that input stream and returns a result. Hence any kind of intermediate download is not needed and removes a good bit of boiler plate code for closing and reading from streams.